### ISSUE TITLE
Remove whitespace under avatars on the left

### ISSUE THREAD

**Issue by placoderm** [tuskyapp/Tusky#2807]
Thanks for making the app! Looks great so far.

One suggestion. Phones are small (for some of us) and the big whitespace (in yellow below) feels like a big waste.
![2022-11-12 11_06_20-Tusky - Mastodon client for Android — Mozilla Firefox](https://user-images.githubusercontent.com/31570884/201459161-4fb3fe99-05ba-4fc0-a60b-443a9f5a4c0b.png)

At the very least, if you want to keep that space, I'd suggest having the avatars be sticky when they hit the top of the screen while that post is still in view. That way the empty space still serves a purpose. 

So it would look like this:
![image](https://user-images.githubusercontent.com/31570884/201459337-9410944d-a090-4a1a-9631-254c4f9d6c9c.png)

* * * *
- Tusky Version: 19.0
- Android Version: ?
- Android Device: LG
- Mastodon instance (if applicable):

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by nikclayton** [comment]
I suggest moving the post view so it starts immediately under the user's avatar.

This frees up an extra line of content to the right of the avatar. We could display the user's address there (so the first line is the name, second line is the address). This probably solves https://github.com/tuskyapp/Tusky/issues/2918 and https://github.com/tuskyapp/Tusky/issues/2816 too.

The extra line taken up by moving the post text is probably recovered in most cases by the extra width to show the post, so I expect this will result in posts taking less vertical space overall.

At the moment the space under the avatar is used to draw a vertical line between posts that are part of the same thread. That could be replaced with e.g. a shorter line that connects the bottom of one post with the top of the the next.

**Comment by Lakoja** [comment]
Was thinking a bit about this and was initially inclined to refuse it - that whitespace serves a bit of a marker what a post is (or at least in conjunction with the avatar it does.

However the other view also already exists: in the conversation view.
And there, I think, it looks good as well.

So now I tend to agree.
Just add a bit more margin between posts and it should work as well.
Furthermore it would be consistent with the conversation view then..

**Comment by mcclure** [comment]
I absolutely do not want this so please don't make this change unless there's a way to turn it off.

Many, many apps do the "avatar column" thing on purpose. It is a way of visually helping the user to distinguish items in a list. This very github view I am typing into right now does it. It's true the conversation view omits the avatar column for the top entry, and that does look good, but the reason it works is because the top item in the conversation view is not an item in a list.

**Comment by LordHDL** [comment]
I'm also in favor of removing the extra space. It's a way to reclaim space that should deservedly go to the content of posts and just looks cleaner.

With respect to distinguishing things in a list, this is already done via faint lines, usernames, and the buttons at the bottom of posts. Both the web version and official Mastodon app use this behavior and the divisions are plenty clear.

That said, I would agree with the option to toggle between the two views.

**Comment by placoderm** [comment]
> It is a way of visually helping the user to distinguish items in a list.

I'd say that the app, at the very least, does a poor job of this with the big white space. At a minimum it it is going to have the white space, then it should cause the user avatar to stick to the top, that way you are at least getting the full benefit of the wasted space. As it is, when you scroll past the top of the post, all you have is white space which tells you nothing.



> That said, I would agree with the option to toggle between the two views.

Something as basic as this should not have a setting. Once you start making things like this optional it makes the app design much more complicated because things have to always look good in both options.

**Comment by Tak** [comment]
I agree with @mcclure - removing the indentation would make the timeline feel like a huge wall of content with no differentiation

**Comment by nikclayton** [comment]
Here's a live example I've been running with for the last day or so.

<img width="50%" src="https://user-images.githubusercontent.com/773100/206864459-20666380-5180-46dc-ac99-5f3b3acfc90f.png">

**Comment by awebeer256** [comment]
Any progress on this? It's a dealbreaker for me: I won't even consider making Tusky my main fediverse app until this change is made. The text area is *far* too narrow.

**Comment by Offerel** [comment]
I can understand that this is a controversial point. But maybe we can get this as an optional setting. I really want the post to use the full width of the display.
